src/utils/axios-auth.ts

```diff
@@ -1,23 +1,24 @@
-import Axios from 'axios'
+import axios from 'axios'
```

Comment:
Small changes to keep this in sync with Filings UI.

Reply:
But now that I look at it, maybe the previous code was better... thoughts?

Reply:
i like the changes.  Might be adding an additional line for `kcToken` but I like it for debugging and it's easy to read.  I'm sure there are others who will thinking the previous way is easier to read 😀